<launch>

	<!-- START A GROUP WITH A NAMESPACE -->
	<group ns="asc">

		<!-- LAUNCH A "I2C for Servos" NODE -->
		<node
			pkg    = "asclinic_pkg"
			name   = "i2c_for_servos"
			output = "screen"
			type   = "i2c_for_servos"
		/>

	</group>

</launch>
